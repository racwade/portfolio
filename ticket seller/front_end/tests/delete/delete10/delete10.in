login
test_selling
delete
simpleUsername
logout