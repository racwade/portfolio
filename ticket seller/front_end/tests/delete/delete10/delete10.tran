02 test_sell      SS 000009999