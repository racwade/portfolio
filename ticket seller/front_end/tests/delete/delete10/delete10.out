welcome to the ticket seller system
enter username:
logged in as test_selling
enter account username:
error: Please log in as admin to use this service.
bye